src/applications/vaos/new-appointment/components/ContactInfoPage.jsx

```diff
@@ -140,7 +150,9 @@ export default function ContactInfoPage() {
     bestTimeToCall: {
       'ui:title': 'What are the best times for us to call you?',
       'ui:validations':
-        !featureAcheronService && flowType === FLOW_TYPES.REQUEST
+        (!featureAcheronService && flowType === FLOW_TYPES.REQUEST) ||
+        (featureAcheronService &&
+          userData.facilityType === FACILITY_TYPES.COMMUNITY_CARE)
```

Comment:
This ensures validation under best time to call for community care requests when acheron service is on